README.md

```diff
@@ -65,3 +65,4 @@ These publications cover many of the internal components used in Chipyard. Howev
 [firemarshal]: https://github.com/firesim/FireMarshal/
 [ariane]: https://github.com/pulp-platform/ariane/
 [gemmini]: https://github.com/ucb-bar/gemmini
+[nvdla]: https://nvdla.org/
```

Comment:
Beware that the `https` NVDLA site will give you a certificate warning. I've always had to use the `http` website unfortunately.

Reply:
I just blindly used the `https`. Ill go ahead and update this.